add_ia_covers/fixed_books.csv

```diff
@@ -0,0 +1,420 @@
+OL10157477M
```

Comment:
Let's write here how this is for books taken from the add_ia_covers (or wherever it comes from) and covers are added to them. Also, it'll be good to write if this is done manually or automatically.

Reply:
This is a data file. The CSV format doesn't support comments.